base/src/main/java/com/smartdevicelink/managers/screen/menu/BaseMenuManager.java

```diff
@@ -780,17 +780,7 @@ private void setAddStatus(Integer size, List<Integer> newArray){
 
 	@SuppressWarnings("BooleanMethodIsAlwaysInverted")
 	private boolean supportsImages(){
-		if (defaultMainWindowCapability != null && defaultMainWindowCapability.getImageFields() != null) {
-			List<ImageField> imageFields = defaultMainWindowCapability.getImageFields();
-			if (imageFields != null && !imageFields.isEmpty()) {
-				for (ImageField field : imageFields) {
-					if (field != null && field.getName() != null && field.getName().equals(ImageFieldName.cmdIcon)) {
-						return true;
-					}
-				}
-			}
-		}
-		return false;
+		return (defaultMainWindowCapability == null || defaultMainWindowCapability.getImageFields() == null) || ManagerUtility.WindowCapabilityUtility.hasImageFieldOfName(defaultMainWindowCapability, ImageFieldName.cmdIcon);
```

Comment:
Same question before around if the `imageField` array is null. @joeljfischer for visibility.